app/controllers/api/v1/articles_controller.rb

```diff
@@ -2,12 +2,23 @@ class Api::V1::ArticlesController < ApplicationController
   rescue_from ActiveRecord::RecordNotFound, with: :article_not_found
 
   def index
-    if params[:category]
-      articles = Article.where(category: params[:category]).paginate(page: params[:category], per_page: 4)
+    if params[:location]
+      articles = Article.where(:location => params[:location]).paginate(page: params[:page], per_page: 4)
+      if params[:category]
+        articles = articles.where(category: params[:category]).paginate(page: params[:category], per_page: 4)
```

Comment:
`page:` should be `params[:page]` (old error?)

Reply:
yeah, missed that, from pulled code